src/gui/src/AppConfig.cpp

```diff
@@ -243,3 +245,44 @@ bool AppConfig::getAutoStart() { return m_AutoStart; }
 void AppConfig::setMinimizeToTray(bool b) { m_MinimizeToTray = b; }
 
 bool AppConfig::getMinimizeToTray() { return m_MinimizeToTray; }
+
+void AppConfig::setStartWithComputer(bool b)
```

Comment:
This needs to go in via separate PR.

Reply:
Yep, I'll split it out, good call. One moment.